hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -1139,13 +1139,33 @@ def binary(*, min_size: int = 0, max_size: int = None) -> SearchStrategy[bytes]:
 
 @cacheable
 @defines_strategy
-def randoms() -> SearchStrategy[random.Random]:
-    """Generates instances of ``random.Random``, tweaked to show the seed value
-    in the repr for reproducibility.
-
-    Examples from this strategy shrink to seeds closer to zero.
+def randoms(
+    *, silent: bool = True, instrument: bool = None
```

Comment:
Maybe "verbose = False" instead of "silent = True"? "Instrument" doesn't convey the right meaning to me - I'd expect instrumenting something to provide more output, not change its behaviour. Maybe `true_random` with the sense inverted?

Reply:
`verbose` is ambiguous with other usage of that (in settings) I think, but maybe something more explicit like `note_results`. I like `true_random`. Maybe `use_true_random` to keep the imperative tone?